projects/canopy/src/styles/variables/components/button/_button.scss

```diff
@@ -5,5 +5,9 @@
   --btn-disabled-border-color: var(--disabled-color);
   --btn-disabled-color: var(--color-taupe-grey);
 
+  --btn-padding-horizontal: 0.6875rem; // 11
```

Comment:
Just looking at the button padding before this PR... it was 

`padding: var(--space-xs);`

If it was previously important to be using a standardised spacing value it feels strange to be discarding that especially for the sake of a pixel... i'm not sure what the right thing to do here is but I think its a good discussion point before it potentially becomes widespread

Reply:
The initial t-shirt sizing approach was mostly a development led idea during the early stages. The design team have been pretty happy to adopt it as it all conforms to the 4px design grid, I think it generally works well for things like layout and spacing. However when we get down to the slightly fiddle detail like inside the buttons I've been happy to concede that we might need a bit more granularity. We can keep an eye on it though and try and keep to the pre-defined spaces where possible

Reply:
yeah your right it might be one for further conversations. it might just be the disconnect between our lint rules for using rem and the practice of contorting them to work with a pixel based design that seems strange. either way it's not a show stopper as we are already doing it to a certain degree in the code base